example/src/App.js

```diff
@@ -1,11 +1,11 @@
-import React, { Component } from 'react'
+import React, { Component } from "react"
```

Comment:
It's better to use only `import * as React from 'react'` instead of mixing named and default imports.